### ISSUE TITLE
brightness-widget: attempt to call a nil value

### ISSUE THREAD

**Issue by senorsmile** [streetturtle/awesome-wm-widgets#68]
![image](https://user-images.githubusercontent.com/2666849/52524783-d4783200-2c55-11e9-8ec1-9bb423a5b613.png)

I get this pop up error whenever I change the brightness.  The brightness change is working fine, but the error is annoying.  A quick glance over the lua doesn't have anything sticking out to me.  

Line 50 looks like the main function.  Here are lines 49-51: 

```lua
    spawn.easy_async(GET_BRIGHTNESS_CMD, function(stdout, stderr, exitreason, exitcode) 
        update_widget(brightness_widget, stdout, stderr, exitreason, exitcode)
    end)
```

**Comment by streetturtle** [comment]
Thanks for raising an issue :+1: 
I've faced the same thing when tried to change the brightness by mouse. 

TL;DR I've pushed the fix - you can pull the changes or just remove the code block you've pasted and slightly change the command to `light -U 5` and `light -A 5`.

The problem was in the code block you pasted - there are two places where the widget is updated:
 -  in the `watch` function at the bottom, which simply executes the command to get brightness every second, and 
 - in the code block you've pasted, every time key mouse scrolls it runs the command, if you scroll too fast it brakes for some reason, probably there is some race condition happens. So it's better to remove that code block as most likely we don't need that level of responsiveness of the widget.

**Comment by streetturtle** [comment]
I guess it should be ok now, feel free to reopen it, I would be more than happy to help. Thanks again for raising an issue :+1:

**Comment by senorsmile** [comment]
Thanks.  It works!
